### PR TITLE
guilds proposal page mobile

### PR DESCRIPTION
**Pull request by luchux**
Done: 
- added the media queries for the mobile view of proposal page in Guilds. 
- modify some styles as in the figma files. 

screencapture-localhost-3000-2021-11-30-10_54_59

### CODE REVIEW

File: src/components/Guilds/ProposalStatus/index.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import styled from 'styled-components';
 import { Box } from '../common/Layout';
+
 const Status = styled.div`
   display: flex;
   flex-direction: row;
   width: 100%;
-  justify-content: flex-end;
+  justify-content: flex-start;
```
Comment (hhamud) [inline src/components/Guilds/ProposalStatus/index.tsx]: changing this value to start, is causing this issue on the proposals page, where the status button is squishing up the text
image

**Review by hhamud — CHANGES_REQUESTED** [review]
The proposal looks great but for some reason the changes have affected the proposals page as well here.

File: src/components/Guilds/ProposalStatus/index.tsx (hunk 1 above)
Comment (luchux) [reply src/components/Guilds/ProposalStatus/index.tsx]: nice!

**Review by hhamud — CHANGES_REQUESTED** [review]
You fixed the previous issue with the status button but now there's a new issue here that seems to have broken. 
image

**Review by hhamud — APPROVED** [review]

**Pull request merged**